package.json

```diff
@@ -3,25 +3,24 @@
   "version": "0.0.5",
   "description": "Hapi.js plugin for authentication scheme of accepting connections only from certain IPs",
   "main": "lib/index.js",
-  "typings": "lib/index.d.ts",
   "repository": "https://github.com/chamini2/hapi-auth-ip-whitelist",
   "author": "Matteo Ferrando <matteo.ferrando2@gmail.com>",
   "license": "MIT",
-  "scripts": {
-    "build": "tsc",
-    "prepublish": "npm run build"
-  },
   "devDependencies": {
-    "@types/boom": "^4.3.7",
-    "@types/hapi": "^16.1.12",
-    "@types/ip": "^0.0.30",
-    "typescript": "^2.6.2"
+    "hapi": "^17.2.0",
+	"therealyou": "^2.0.0"
   },
   "dependencies": {
-    "boom": "^5.2.0",
+    "boom": "^7.1.1",
     "ip": "^1.1.5"
   },
   "peerDependencies": {
-	"hapi": "< 17"
+    "hapi": ">= 17"
+  },
+  "optionalDependencies": {
+	"therealyou": "^2.0.0"
+  },
+  "engines": {
+	"node": ">=8.9.0"
```

Comment:
Why this node?

Reply:
It's the node version specified in package.json for Hapi v17. Doesn't make sense to support older node versions than the base module that makes use of this plugin.

Reply:
I think that `engines` es specific for **our** needs, not our dependencies'. We have code that can be run in lower versions of node.

Imagine if Hapi v17 becomes compatible with more `node` versions and `ip-whitelist` throws a warning even though is not true that we only work in `8.9.0`. We should check what is the real `node` version we need (independent of Hapi) and set it to that or drop the `"engines"` property.

This is part of why Typescript is good, it lets you write code that is then translated into ES5.